#
#  Initial Speed Profile
#

ros_whill:
  ros__parameters:
    init_speed:
      forward:
        speed: 0.500  # [m/s]
        acc: 0.23     # [m/ss]
        dec: 1.25     # [m/ss]
      backward:
        speed: 0.600  # [m/s]
        acc: 0.34     # [m/ss]
        dec: 0.89     # [m/ss]
      turn:
        speed: 0.8    # [rad/s]
        acc: 0.79     # [rad/ss]
        dec: 1.8      # [rad/ss]

